cogs/games.py

```diff
@@ -1,7 +1,11 @@
 import random
-
+import discord
 from discord.ext import commands
 
+from assets.context import Command, argument, example, usage_info
+from cogs.command_utils import games_util
+from typing import Union
```

Comment:
C0411: Standard import "from typing import Union" should be placed before "import discord"